compiler/rustc_trait_selection/src/traits/fulfill.rs

```diff
@@ -496,6 +496,13 @@ impl<'a, 'b, 'tcx> FulfillProcessor<'a, 'b, 'tcx> {
                         obligation.cause.span,
                     ) {
                         Ok(()) => ProcessResult::Changed(vec![]),
+                        Err(ErrorHandled::TooGeneric) => {
```

Comment:
I don't know anything about this part of the compiler...

Reply:
:+1:

yeah, then let's have @varkor or @eddyb review that part